tests/components/radarr/test_calendar.py

```diff
@@ -0,0 +1,41 @@
+"""The tests for Radarr calendar platform."""
+from datetime import timedelta
+
+from freezegun.api import FrozenDateTimeFactory
+
+from homeassistant.components.radarr.const import DOMAIN
+from homeassistant.const import STATE_OFF, STATE_ON
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+from . import setup_integration
+
+from tests.test_util.aiohttp import AiohttpClientMocker
+
+
+async def test_calendar(
+    hass: HomeAssistant,
+    aioclient_mock: AiohttpClientMocker,
+    freezer: FrozenDateTimeFactory,
+) -> None:
+    """Test for successfully setting up the Radarr platform."""
+    freezer.move_to("2021-12-02 00:00:00-08:00")
+    entry = await setup_integration(hass, aioclient_mock)
+    coordinator: DataUpdateCoordinator = hass.data[DOMAIN][entry.entry_id]["calendar"]
```

Comment:
Don't access hass.data or the coordinator in tests. Those are details.

https://developers.home-assistant.io/docs/development_testing#writing-tests-for-integrations